src/filefuncs.c

```diff
@@ -46,6 +46,9 @@ static char **getfiles(const char *fspec, int *nfound, LOG_INFO* LogInfo) {
 
 	DirName(fspec, dname); // Copy `fspec` into `dname`
 	fname = Str_Dup(BaseName(fspec), LogInfo);
+    if(LogInfo->stopRun) {

```

Comment:
In `getfiles()`, we need to move `(*nfound) = 0;` (now line 63) to before the first early return on error (now line 50). Otherwise, `RemoveFiles()` will get whatever value for `nfiles` when it enters `for (i = 0; i < nfiles; i++) Mem_Free(flist[i]);` and possibly fails.